### PR TITLE
Add `shuffle` implementation to preserve stability

### PR DESCRIPTION
**Pull request by ericphanson**
xref https://github.com/JuliaRandom/StableRNGs.jl/issues/10

### CODE REVIEW

File: src/StableRNGs.jl [hunk 1]
```diff
@@ -132,5 +139,20 @@ for T in Base.BitInteger_types
         SamplerRangeFast(r)
 end
 
+# https://github.com/JuliaRandom/StableRNGs.jl/issues/10
+Random.shuffle(r::StableRNG, a::AbstractArray) = Random.shuffle!(r, Base.copymutable(a))
```
Comment (rfourquet) [inline src/StableRNGs.jl:143]: Is this one necessary?

**Review by rfourquet — APPROVED** [review]
Thanks!

File: src/StableRNGs.jl (hunk 1 above)
Comment (ericphanson) [reply src/StableRNGs.jl:143]: I'm not sure, but I would prefer to keep it, in case it becomes necessary. It doesn't seem harmful either.

**Comment by gdalle** [comment]
@rfourquet would this deserve a new release?

**Comment by ericphanson** [comment]
It got one: https://github.com/JuliaRegistries/General/pull/96886

**Comment by gdalle** [comment]
My bad, I just checked out the repo tags and the last one is from 2020. Probably because TagBot is not installed?

**Comment by rfourquet** [comment]
Thanks for noticing @gdalle, I've fixed this and made a PR to add TagBot (#21). 
@ericphanson I invited you to the repo, hoping you don't mind!

**Pull request merged**